style.css

```diff
@@ -0,0 +1,1783 @@
+* {
+  margin: 0;
+  padding: 0;
+  touch-action: auto;
+}
+
+/* This is the containers where we are center over project */
+.container-1 {
+  display: flex;
+  justify-content: center;
+  align-content: center;
+}
+
+/* Giving the heigh */
+.container {
+  height: 718px;
+  width: 375px;
+}
+
+.nav-right {
+  display: none;
+}
+
+.main-img-d {
+  display: none;
+}
+
+/* Design the navbar */
+.navbar-1 {
+  background-color: #fff;
+  height: 92px;
+  width: 100%;
+  display: flex;
+  font-family: "Poppins", sans-serif;
+  font-weight: 700;
+  justify-content: space-between;
+  align-items: center;
+  color: #6070ff;
+  font-size: 18px;
+}
+
+.navbar-1 a {
+  text-decoration: none;
+  list-style: none;
+}
+
+.navbar a {
+  text-decoration: none;
+  color: #6070ff;
+}
+
+.card a {
+  text-decoration: none;
+}
+
+.lang-1 a {
+  display: grid;
+  padding: 7px;
+  background: #f7f7f9;
+  grid-template-columns: 8vh auto auto;
+}
+
+.navbar a:hover {
+  color: #6070ff;
+}
+
+.navbar a:active {
+  color: #2230d2;
+}
+
+/* This is logo  */
+.navbar .left {
+  text-align: center;
+  list-style: none;
+  flex-basis: 100px;
+  background-color: #fff;
+}
+
+/* This is the icon area */
+
+.navbar .right {
+  list-style: none;
+  flex-basis: 40px;
+  background-color: #fff;
+}
+
+.material-symbols-outlined {
+  background-color: #fff;
+}
+
+/* Add bacgorund and also do flex as column wise */
+.main-text {
+  height: 100%;
+  background-color: #fff;
+  background-image: url(./Microverse-Student-Project-1/header-shapes\ mobile.png);
+  border-radius: 0 0 0 56px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 0 21px 0 21px;
+  gap: 7px;
+}
```

Comment:
- [ ] Please make sure that your background image weight is not larger than the window size. kindly reduce and fix it. you can add `background-repeat: no-repeat, background-size: cover;`
### Your design👇
![stpro](https://user-images.githubusercontent.com/66992901/200008564-144fcc9d-3139-4d1b-8587-b4b3456fcf89.PNG)